api/__tests__/routes/apiKeyMiddleware.test.js

```diff
@@ -0,0 +1,61 @@
+const axios = require('axios');
+const eventsHelper = require('../../helpers/events');
+const { apiKeyMiddleware } = require('../../routes/apiKeyMiddleware');
+axios.defaults.adapter = require('axios/lib/adapters/http');
+process.env = {
```

Comment:
There's a risk with this that it could mess up other tests that also rely on `process.env` as you're altering the value here but not resetting it after the test.  To avoid that:

- Move this line here (where you set `process.env`) into inside each of your two tests below
- Add lines 7-16 from [this file](https://github.com/Scottish-Tech-Army/Volunteer-app/blob/develop/api/__tests__/services/logging.test.js#L7) after the `describe(...` line below and before the first `test(...` -- this will reset `process.env` before/after your tests